bindings/nodejs/examples/how_tos/native_tokens/send.ts

```diff
@@ -14,7 +14,7 @@ const RECV_ADDRESS =
 // In this example we will send native tokens.
 //
 // Make sure that `STRONGHOLD_SNAPSHOT_PATH` and `WALLET_DB_PATH` already exist by
-// running the `how_tos/accounts-and-addresses/create-wallet` example!
+// running the `how_tos/accounts-and-addresses/create-account` example!
```

Comment:
`accounts-and-addresses` is supposed to have underscores, not dash

Reply:
good point, fixed